src/nvhttp.cpp

```diff
@@ -278,17 +282,17 @@ namespace nvhttp {
       if (x == 2) launch_session.fps = atoi(segment.c_str());
       x++;
     }
-    launch_session.unique_id = (get_arg(args, "uniqueid"));
-    launch_session.uuid = (get_arg(args, "uuid"));
-    launch_session.appid = util::from_view(get_arg(args, "appid"));
-    launch_session.enable_sops = util::from_view(get_arg(args, "sops"));
-    launch_session.surround_info = util::from_view(get_arg(args, "surroundAudioInfo"));
-    launch_session.gcmap = util::from_view(get_arg(args, "gcmap"));
+    launch_session.unique_id = (get_arg(args, "uniqueid", "0123456789ABCDEF"));
+    launch_session.uuid = (get_arg(args, "uuid", "00000000000000000000000000000000"));
+    launch_session.appid = util::from_view(get_arg(args, "appid", "0123456789"));
```

Comment:
I don't think we should provide anything except an empty string as the default.

```suggestion
    launch_session.unique_id = (get_arg(args, "uniqueid", ""));
    launch_session.uuid = (get_arg(args, "uuid", ""));
    launch_session.appid = util::from_view(get_arg(args, "appid", ""));
```

Reply:
Sure, that works. I'm not familiar with the internal workings of Sunshine and wanted to make sure the provided defaults at least somewhat represent the actual values as to not trip up any logic elsewhere. You're probably more familiar with the risks.

Reply:
or `"None"`, `"unknown"`... something like that

Reply:
I changed them to `"unknown"`. Does that work?